api/src/main/scala/dcos/metronome/api/v1/controllers/LaunchQueueController.scala

```diff
@@ -0,0 +1,13 @@
+package dcos.metronome.api.v1.controllers
+
+import dcos.metronome.api.{ Authorization, RestController }
+import dcos.metronome.queue.LaunchQueueService
+import dcos.metronome.api.v1.models.QueuedJobRunMapWrites
+import play.api.mvc.Action
+
+class LaunchQueueController(queueService: LaunchQueueService) extends RestController {
+
+  def queue() = Action {
+    Ok(QueuedJobRunMapWrites.writes(queueService.listGroupByJobId()))
```

Comment:
let's do `.groupBy(_.jobid)` here as we do in LaunchQueueService and then we can drop the listGroupByJobId altogether...

Reply:
I debated this... but I like it!   Thanks!